modular_skyrat/modules/mentor/code/mentorhelp.dm

```diff
@@ -38,7 +38,7 @@
 		else
 			.["present"]++
 
-/proc/key_name_mentor(whom, include_link = null, include_name = FALSE, include_follow = FALSE, char_name_only = FALSE)
+/proc/key_name_mentor(whom, include_link = null, include_name = FALSE, include_follow = FALSE, char_name_only = FALSE, message)
```

Comment:
That's just not how that's meant to be used.

Reply:
without big re-working on mentor system it's easier way to get message, which used in this proc

Reply:
anyway.. removed